packages/markdown/components/Embed.jsx

```diff
@@ -0,0 +1,86 @@
+const React = require('react');
+const propTypes = require('prop-types');
+const Embedly = require('embedly');
+
+const api =
+  process.env.NODE_ENV !== 'test'
+    ? new Embedly({ key: 'f2aa6fc3595946d0afc3d76cbbd25dc3' })
```

Comment:
I agree that hitting the Embedly API from the front end is less than ideal—this is just a short term solution to paper over the fact that we store the Embedly results directly in our magic block JSON. @domharrington and I have discussed this as a stopgap measure, until we have a better way of storing/retrieving this info. (Probably something we'll have to address as we build out the new editor.

Reply:
Sorry, I didn't notice that this file wasn't a test so ignore that part of my comment.

Regarding the API key itself, my main concern here is that we're storing an API key in a public repository. At the very least, this should be a passed-in configuration.

Reply:
I can get this compiling from a `.env` file (by using the dotenv package in readme/markdown), but the key is still compiled in to the final `dist/` build so I'm not really sure that can be considered a solution. @domharrington do you have any ingenious ideas as to avoid including this Embedly key in the repo?